core/domain/subtopic_page_domain_test.py

```diff
@@ -23,6 +23,80 @@
 import utils
 
 
+class SubtopicPageChangeDomainUnitTests(test_utils.GenericTestBase):
+    """Tests for subtopic page change domain objects."""
+    user_id = 'user_id'
+    topic_id = 'topic_id'
+    subtopic_id = 1
+
+    def setUp(self):
+        super(SubtopicPageChangeDomainUnitTests, self).setUp()
+        self.subtopic_page_change = subtopic_page_domain.SubtopicPageChange({
+            'cmd': subtopic_page_domain.CMD_UPDATE_SUBTOPIC_PAGE_PROPERTY,
+            'subtopic_id': 1,
+            'property_name': 'page_contents_html',
+            'new_value': 'a',
+            'old_value': 'b'
+            })
+
+    def test_init_cmdnotindict_raisesexception(self):
+        with self.assertRaisesRegexp(Exception, 'Invalid change_dict *'):
```

Comment:
Add a step to that ensures changes with cmd doesn't raise an exception. (see 1st bullet of this [wiki page](https://github.com/oppia/oppia/wiki/Writing-backend-tests#guidelines-for-writing-good-tests))

Take a look at "baseline validation" and follow this pattern for all other tests.